Ports/lua/oslibc.patch

```diff
@@ -0,0 +1,20 @@
+--- lua/src/loslib.c	2017-04-19 13:29:57.000000000 -0400
++++ /home/mustafa/src/lua/src/loslib.c	2019-05-28 05:44:29.523310628 -0400
+@@ -127,7 +127,7 @@
+ #else				/* }{ */
+ 
+ /* ISO C definitions */
+-#define LUA_TMPNAMBUFSIZE	L_tmpnam
++#define LUA_TMPNAMBUFSIZE	tmpnam
```

Comment:
The POSIX standard says that `L_tmpnam` should be defined by `stdio.h`:
http://pubs.opengroup.org/onlinepubs/9699919799/basedefs/stdio.h.html

I think a better fix would be to edit Serenity's `LibC/stdio.h` to include something like:
```
#define L_tmpnam 256
```

With that fix, you won't need this Lua patch at all :)